triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/RunnerConfig.scala

```diff
@@ -249,6 +249,16 @@ object RunnerConfig {
           throw new IllegalArgumentException(s"Unsupported logging encoder $other")
       }
 
+    opt[Long]("max-batch-size")
+      .optional()
+      .text(
+        s"maximal number of message processed between two high-level rule triggers. Defaults to ${DefaultTriggerRunnerConfig.maximumBatchSize}"
+      )
+      .action((size, cli) =>
+        if (size > 0) cli.copy(triggerConfig = cli.triggerConfig.copy(maximumBatchSize = 1))
```

Comment:
Here (and in `Cli.scala` below), should this be ```if (size > 0) cli.copy(triggerConfig = cli.triggerConfig.copy(maximumBatchSize = size))```?

Reply:
good catch.